seirmo/__init__.py

```diff
@@ -19,3 +20,7 @@
 from .simulation import (
     SimulationController
 )
+
+from .subplot import (
```

Comment:
I would suggest to change the directory structure here a little bit, so that the figures are all in one directory. i.e.
```
seirmo
    apps
    plots
        figures.py
        __init__.py
     tests
     models.py
     simulation.py
     ...
     __init__.py
```
The figures.py should have both figure class or if you prefer you can also have the classes in separate files, but than you'd have to give the files more descriptive names, like _incidence_plot.py and _compartment_plot.py.